experiment/measurer/coverage_printer_utils.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility DataFrameContainer class and functions for report generation on
+segment and function coverage for the entire experiment
+(all Fuzzer-benchmark-trial combinations)."""
+
+import os
+import hashlib
+import pandas as pd
```

Comment:
nit: put a space before this import to seperate stdlib imports from third-party.